book/content/very_large_datasets/very_large_datasets.md

```diff
@@ -0,0 +1,56 @@
+# Sharing Very Large Datasets
```

Comment:
It would be great to have a link to the Research Data Management chapter here highlighting that you describe a fairly specific case of managing data?

Reply:
Sure! 

I'm currently writing up my thesis so I won't be able to come back to this for several weeks. I'm more than happy to have this section included as-is, or to have someone else help me clean up this chapter first. Whatever works for you all!

Reply:
Great, we will see how much capacity we have to work on this. Thanks for the super quick feedback and all the best for your thesis writing!

Reply:
No problem! I'll probably pop in every once in a while as a break from thesis writing ;) but yeah, help with improvements would be great!